Library/Homebrew/cask/cmd/automerge.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require "cask/cmd/abstract_internal_command"
+require "tap"
+require "utils/formatter"
+require "utils/github"
+
+module Cask
+  class Cmd
+    class Automerge < AbstractInternalCommand
+      OFFICIAL_CASK_TAPS = [
+        "homebrew/cask",
+        "homebrew/cask-drivers",
+        "homebrew/cask-eid",
+        "homebrew/cask-fonts",
+        "homebrew/cask-versions",
+      ].freeze
+
+      def run
+        raise "This command should only be run by Homebrew maintainers." unless ENV["HOMEBREW_DEVELOPER"]
```

Comment:
```suggestion
        raise "This command may only be run by Homebrew maintainers." unless ENV["HOMEBREW_DEVELOPER"]
```

I just know non-maintainers will try to run this, and be puzzled. “may” instead of “can” because non-maintainers can still run technically it, it just won’t work.